app/src/test/java/com/example/parkingsystem/activities/MenuActivityTest.java

```diff
@@ -0,0 +1,7 @@
+package com.example.parkingsystem.activities;
+
+import junit.framework.TestCase;
+
+public class MenuActivityTest extends TestCase {
```

Comment:
adding empty classes could potentially have a bad impact on code maintainability as this could be a unused class forever

Reply:
I was unable to find that class in my code, please let me know if you are still able to see it after last commit.